packages/blocks/table/src/components/filter-detail.tsx

```diff
@@ -126,14 +133,10 @@ export const FilterDetail: React.VFC<FilterDetailProps> = ({
 
   if (multiFilter && !filters.length && !isMounted.current) {
     isMounted.current = true;
-    const fieldsWithId = (multiFilter.filters ?? []).map(
-      ({ field, value, operator }) => ({
-        field,
-        value,
-        operator,
-        id: uuid(),
-      }),
-    );
+    const fieldsWithId = (multiFilter.filters ?? []).map((filter) => ({
+      ...filter,
+      id: uuid(),
```

Comment:
Just curious, why do these filters each have ids in the table block? Just for the purpose of using them in the react key?

Reply:
I'm not sure - @teenoh ?

Reply:
we need the ids to locate a particular filter from the list of filters and update it's value when there's a change @benwerner01